reference/activity_manager/project_completeness.rst

```diff
@@ -0,0 +1,102 @@
+What is the project completeness?
+=================================
+
+Goal
+____
+
+The Activity Manager feature is about creating project from a products selection that give vision to contributors (users
+in charge of enrichment). Each project gives to the contributor its own completeness on the products selection, taking
+in account its permissions to edit or not products, attribute groups, locales, scopes.
+
+We need to go deeper
+____________________
+
+The need is to apply permissions of each user on a products selection in order to acquire a new sub-selection. Then
+calculate which percentage of this sub-selection is not started to enrich, in progress or done. That is to say for each
+user and for each product in the selection apply permissions and in the user perimeter watching if there is at least one
+attribute filled or all attributes filled.
```

Comment:
I don't get the purpose of this whole block, you loose me as a reader. Maybe you can just resume to 

> "As every user will have his very own completeness depending on his rights, we need to compute every product completeness for every user... As you can guess, it's a really huge task that cannot be done synchronously.